css/custom.css

```diff
@@ -17,8 +17,16 @@
   text-align: center;
   text-decoration: none;
 }
+.ti-google{
```

Comment:
Work on one issue only in one PR.

Reply:
umm excuse me?

Reply:
The title referred to adding meta tags for Apple then why are you changing the CSS classes?

Reply:
oh lemme remove that. 
thanks